ios/browser/api/networking/models/extensions/BraveCertificateProxyCertInfoExtensionModel.mm

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "BraveCertificateProxyCertInfoExtensionModel.h"
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #import "brave/ios/browser/api/networking/common/brave_certificate_enums.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_ios_utils.h"
+  #include "brave/ios/browser/api/networking/utils/brave_certificate_utils.h"
+#else
+  #import "brave_certificate_enums.h"
+  #include "brave_certificate_ios_utils.h"
+  #include "brave_certificate_utils.h"
+#endif
+
+#if defined(BRAVE_CORE) // Compiling in Brave-Core
+  #include "third_party/boringssl/src/include/openssl/x509.h"
+  #include "third_party/boringssl/src/include/openssl/x509v3.h"
+#else
+  #include <openssl/x509.h>
+  #include <openssl/x509v3.h>
+#endif
+
+
+@implementation BraveCertificateProxyPolicyExtensionModel
+- (void)setLanguage:(NSString*)language {
+  _language = language;
+}
+
+- (void)setPolicyText:(NSString*)policyText {
+  _policyText = policyText;
+}
+@end
+
+@implementation BraveCertificateProxyCertInfoExtensionModel
+- (void)parseExtension:(X509_EXTENSION*)extension {
+  _pathLengthConstraint = -1; //infinite
```

Comment:
Do you actually want -1 or do you want `NSIntegerMax` (based on comment saying infinite I'm unsure of intention)

Reply:
I want -1 as that's what's in the spec. I updated the comment to reflect it :)